### ISSUE TITLE
Empty "Received Output"

### ISSUE THREAD

**Issue by Brzystek** [agrawal-d/cph#45]
**Extension Version:** 4.0.3
**VS Code Version:** 1.46.1

Your program looks great, but it doesn't work on my vscode with Python. 
![empty_output](https://user-images.githubusercontent.com/67138199/85102413-e6cc8d80-b204-11ea-86e0-cb36f74b362f.jpg)
When I put "Run All", field "Received Output" is empty. 
When I click "Run Testcases" still is empty. 
When I put Ctrl+Shift+B I see "No bulid taks to run".

**Comment by agrawal-d** [comment]
Hmm, I'm able to repro the error, albeit a bit differently:
![image](https://user-images.githubusercontent.com/7714968/85429168-f862bc00-b59b-11ea-86c8-3d6ff5ea446b.png)

I'll try to find what's the cause.
Btw, the shortcut is `Ctrl+Alt+B`, not `Ctrl+Shift+B`

**Comment by julianferres** [comment]
Same on Python

**Comment by luctivud** [comment]
> Hmm, I'm able to repro the error, albeit a bit differently:
> ![image](https://user-images.githubusercontent.com/7714968/85429168-f862bc00-b59b-11ea-86c8-3d6ff5ea446b.png)
> 
> I'll try to find what's the cause.
> Btw, the shortcut is `Ctrl+Alt+B`, not `Ctrl+Shift+B`

Hey I'm facing the same issue which you were able to repro. Any Update?
I'm on a linux machine. I think the problem is with linux only because it works fine when I checked it on windows10.

**Comment by Brzystek** [comment]
I have this problem on two different computers with Windows 10 (64 bit).

**Comment by jain-aayush1123** [comment]
![image](https://user-images.githubusercontent.com/50782898/89731437-a222dd00-da64-11ea-968f-2d3b124434ca.png)

facing the same issue on windows

**Comment by msnitish** [comment]
Facing the same issue. Doesn't give any output.
Machine : Mac OSX
<img width="916" alt="Screenshot 2020-08-12 at 19 26 11" src="https://user-images.githubusercontent.com/25774996/90023960-f7175b00-dcd1-11ea-9f40-490c6189f856.png">

**Comment by agrawal-d** [comment]
Has the issue been resolved now? Please try updating to the latest version. The error occurs if `python` is not in PATH. ( maybe `python3` is, but the extension does not use that)

**Comment by Brzystek** [comment]
I updated it to the latest version, but it still didn't work.
I checked the PATH and there was a path to the python.
When I opened cmd and run python.exe I saw the Microsoft store. I agreed to install Python from the store (before installing from .exe file). Now I can run the Python with cmd and this extension works. Thank you!

Update: open in Windows "App execution aliases" and switch off all options with python
